Makefile

```diff
@@ -147,6 +146,7 @@ help:
 	@echo "- s3deployprod       Deploys a snapshot specified with SNAPSHOT=xxx to s3 prod."
 	@echo "- s3activateint      Activate a version at the root of a remote bucket. (usage only: make s3activateint S3_VERSION_PATH=<branch>/<sha>/<version>)"
 	@echo "- s3activateprod     Activate a version at the root of a remote bucket. (usage only: make s3activateprod S3_VERSION_PATH=<branch>/<sha>/<version>)"
+	@echo "- s3copybranch       Copy the current directory content to S3 int. Defaults to the current branch name. WARNING: your code must have been compiled with 'make user' first."
```

Comment:
We could execute the user target automatically instead and remove the warning.

Reply:
Yes we could